spec/appendix.md

```diff
@@ -1,12 +1,58 @@
 ## Appendix
 
+### Trust Over IP Glossary - Controlled Terms 
+
+NOTE: 
+* The following set of identifiers are from the current [ToIP Glossary](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit?usp=sharing) (Google Doc). These definitions need to be moved over to the (under development) ToIP Terms tool (aka TEv2).
+* TODO: not all ToIP definitions are fully linked here. This whole section is more of an exemplar as of the time of writing.
+
+[[def: autonomous identifier]] 
+~ Another term for [[ref: self-certifying identifier (SCID)]]. 
+
+[[def: cryptographic keys]]
+~ A key in cryptography is a piece of information, usually a string of numbers or letters that are stored in a file, which, when processed through a cryptographic algorithm, can encode or decode cryptographic data. Symmetric cryptography refers to the practice of the same key being used for both encryption and decryption. Asymmetric cryptography has separate keys for encrypting and decrypting. These keys are known as the public keys and private keys, respectively. Source: Wikipedia.
```

Comment:
Scary to go to this level in a spec of this type. Do you really think this is needed?  That said, if it is needed and you are putting a Source, I suggest including the link to the source.

Reply:
It's a good example of a definition/term that can be left out of this spec and point at the ToIP Glossary. Drummond and I are working with Henk to get the tooling figured. For now I'd prefer to leave it in as we don't have a place to point it. 

This is the entry in the (Google Docs) doc that is a temporary place. 

<img width="681" alt="image" src="https://github.com/trustoverip/tswg-did-method-webs-specification/assets/686855/dc6bb42f-d63d-4ac4-bbbc-e38542e2c409">

Reply:
I would suggest for all of these “defined in the ToIP Glossary” terms, you put the link to the ToIP Glossary.  E.g. for this one use [this link](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit#heading=h.53rzpn1yn6q7) and leave out the reference to Source being Wikipedia.

Or if you don’t want to link to exact header, at least the link to the ToIP Glossary, so we know all the things to be fixed later...

Reply:
resolved in recent push